src/managers/MemberManager.ts

```diff
@@ -21,7 +22,14 @@ export default class MemberManager extends BaseManager<Member, typeof Member> im
     add(ids: string | string[]): Promise<Member> | Promise<Collection<string, Member>> {
         throw new Error('Method not implemented.')
     }
-    remove(member: Member): Promise<this> {
-        throw new Error('Method not implemented.')
+    async remove(member: Member): Promise<FormerMember> {
+        await this.client.rest.api<PostMemberRemoveResponse>(
+            'POST',
+            `groups/${this.group.id}/members/${member.memberID}/remove`,
+        )
+        const formerMember = new FormerMember(this.client, this.group, member.user, member, FormerMemberState.Removed)
+        this.former._upsert(formerMember)
+        this.group.members._remove(member)
```

Comment:
seems a bit redundant to reference `this.group.members` when `this` is already the `MemberManager`
```suggestion
        this._remove(member)
```